src/components/TextInput/TextInput.tsx

```diff
@@ -61,7 +61,7 @@ export type TextInputBaseProps = {
     copyable?: boolean;
     value?: string;
     onChange?: (value: string) => void;
-    onEnterPressed?: (event: KeyboardEvent<HTMLInputElement>) => void;
+    onEnterPressed?: (value: string) => void;
```

Comment:
If no one gave you feedback, you can also search the usage in the web-app yourself to asses the impact of the change.

Reply:
Looks like the prop `onEnterPressed` is used 33% of all `TextInput` implementation in `web-app` as a method handler.  Will need to restructure this to be more versatile

Reply:
@ryancarville What's the status of this PR, Should we close?

Reply:
We can close it actually since we are going to make a new component anyways and this change would be a breaking change. Will keep it in mind for the new one though.